apps/tangle-dapp/containers/NominationsPayoutsContainer/NominationsPayoutsContainer.tsx

```diff
@@ -339,3 +353,45 @@ function ManageButtonContainer(props: {
     </div>
   );
 }
+
+/** @internal */
+function FilterByErasContainer(props: {
+  maxEras: PayoutFilterableEra;
+  setMaxEras: (maxEras: PayoutFilterableEra) => void;
+}) {
+  const { maxEras, setMaxEras } = props;
```

Comment:
Call the Zustand store here to get the state and updater function directly, instead of passing them via props. This will reduce the number of renders when the state updates.